compiler/main/driver.cpp

```diff
@@ -1595,9 +1626,17 @@ static void postprocess_args() {
 
   checkLibraryPythonAndLibmode();
 
-  checkNotLibraryAndMinimalModules();
-
   setPrintCppLineno();
+}
+
+
+// check for things that may be invalid; this happens after
+// 'printStuff()' to avoid having things like 'chpl --help' print
+// errors rather than doing what the user said (for which these
+// checks don't apply because we're not going to compile anything).
+//
+static void checkStuff() {
```

Comment:
`checkStuff()` doesn't seem like a very informative name to me, but I suppose it's no worse than `printStuff()`

Reply:
That is where I took my inspiration from.  :D

Reply:
I renamed this to `validateSettings()` in the most recent commit.